ci-operator/config/insights-qe/iqe-cost-management-plugin/insights-qe-iqe-cost-management-plugin-master__ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: cost-management-interop-aws
+  cron: 0 6 * * 1
```

Comment:
please add support for gangway api to trigger manually.

Reply:
PR is updated.